tests/requirements.txt

```diff
@@ -7,5 +7,5 @@ flake8==3.6.0
 ipdb==0.11
 percy==2.0.1
 pylint==2.3.0
-pytest-dash==2.0.0
+pytest==4.6.4
```

Comment:
Please refer to https://github.com/plotly/dash/blob/dev/requires-testing.txt to remove the redundants

Reply:
And do we still want `--extra-index-url https://test.pypi.org/simple/` (first line of this file)?